src/System.Diagnostics.TraceSource/ref/System.Diagnostics.TraceSource.cs

```diff
@@ -73,6 +74,8 @@ public abstract partial class Switch
         protected string Value { get { throw null; } set { } }
         protected virtual void OnSwitchSettingChanged() { }
         protected virtual void OnValueChanged() { }
+        public System.Collections.Specialized.StringDictionary Attributes { get { throw null; } }
+        virtual protected internal string[] GetSupportedAttributes() { throw null; }
```

Comment:
According to the [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), visibility should be the first modifier, e.g. `protected internal virtual`